medusa/server/api/v2/config.py

```diff
@@ -296,6 +296,13 @@ def data_main():
         section_data['subtitles']['enabled'] = bool(app.USE_SUBTITLES)
         section_data['recentShows'] = app.SHOWS_RECENT
 
+        section_data['default'] = {}
+        section_data['default']['status'] = common.statusStrings[app.STATUS_DEFAULT]
+        section_data['default']['statusAfter'] = common.statusStrings[app.STATUS_DEFAULT_AFTER]
+        section_data['default']['seasonFolders'] = app.SEASON_FOLDERS_DEFAULT
+        section_data['default']['anime'] = app.ANIME_DEFAULT
+        section_data['default']['scene'] = app.SCENE_DEFAULT
```

Comment:
Not sure if default is the most descriptive name. Maybe showDefaults? Open to discussion of course...

Reply:
I think showDefaults is also fine.